src/helpers/helper_raw.rs

```diff
@@ -39,4 +13,49 @@ mod test {
         let r = handlebars.render("t0", &());
         assert_eq!(r.ok().unwrap(), "a{{content}}{{else}}hello");
     }
+    
```

Comment:
You can remove this file and move these tests to template.rs because `{{raw}}` is no longer a helper.